tests/ui/arch/vector_extract_dynamic.rs

```diff
@@ -0,0 +1,11 @@
+// Test `OpVectorExtractDynamic`
+// build-pass
+
+use spirv_std::arch;
+
+#[spirv(fragment)]
+pub fn main() {
+    let vector = glam::Vec2::new(1.0, 2.0);
+    let element = unsafe { arch::vector_extract_dynamic(vector, 1) };
+    assert!(2.0 == element);
```

Comment:
This seems to be missing the `UVec` case from the original.
**EDIT**: ugh there are actually two "originals".